### DISCUSSION TITLE
IBRACORP Video

### DISCUSSION THREAD

**Discussion by ibracorp-sycotix** [Dusk-Labs/dim#409] [Show and tell]
Hi devs and team!

My name's Sycotix from IBRACORP. Just letting you know we'll be dropping a video later today on Dim introducing it to our community and fanbase. We love what you've done and we hope you enjoy the video. You can find our channel here: https://youtube.com/c/IBRACORP

It's an introductory video to hopefully get you some more people testing and providing you feedback!

Developers are invited and are welcome to join our Discord community to receive their respective roles and if they want to collaborate in the future. (https://discord.gg/VWAG7rZ)

Thanks again

**Comment by vgarleanu** [comment]
For those wondering, heres a link to the video: https://www.youtube.com/watch?v=71A10_-eeKM
Thanks again to the IBRACORP team for covering dim!
